speakers-guide.html

```diff
@@ -0,0 +1,266 @@
+---
+layout: subpage-wide
+page_id: speakers-guide
+title: "Speakers Guide"
+subtitle: ""
+permalink: /speakers-guide/
+---
+
+<main>
+  <div class="container">
+    <section>
+      <div class="row">
+        <div class="col-xs-12 col-sm-8 col-sm-offset-2">
+          <p class="intro color--blue">
+            Dear CSSconf EU Speakers,
+          </p>
+          <p>
+            Thanks again for being part of CSSconf EU 2015, we're really looking forward to meeting you in Berlin soon! We created this page to answer any questions you may have, and generally to help make your travels and your stay here as smooth as possible. Please let us know if there's anything that hasn't been addressed here and feel free to contact us any time should you need help or assistance!
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <div class="row">
+        <div class="col-xs-12 col-sm-8 col-sm-offset-2">
+          <h2 class="color--blue">
+            The Team: Here to Help!
+          </h2>
+          <p>Should you have any questions or requests, feel free to speak to us any time!</p>
+          {% include organizers.html %}
```

Comment:
We might want to add some additional `margin-top` for the `.c-team-photos` (will require updating 2017-assets).